wire-ios/Wire-iOS Tests/DeviceManagement/DeviceDetailsViewTests.swift

```diff
@@ -80,21 +80,24 @@ final class DeviceDetailsViewTests: ZMSnapshotTestCase, CoreDataFixtureTestHelpe
                 userClient: client,
                 userSession: mockSession,
                 credentials: emailCredentials,
-                e2eIdentityProvider: e2eIdentityProvider
+                e2eIdentityProvider: e2eIdentityProvider,
+                mlsProvider: mlsProvider
             ),
-            isE2EIdentityEnabled: isE2EIdentityEnabled,
-            isSelfClient: isSelfClient,
             userSession: mockSession,
             getUserClientFingerprint: mockGetUserClientFingerprintUseCaseProtocol,
-            userClient: client
+            userClient: client,
+            isSelfClient: isSelfClient
         )
         viewModel.proteusKeyFingerprint = kFingerPrint
+        viewModel.mlsThumbprint = kFingerPrint
```

Comment:
what does k prefix stands for `kFingerPrint` ?

Reply:
It's just a common convention to indicate that these are constants .